ext/fileinfo/libmagic/apprentice.c

```diff
@@ -632,8 +632,6 @@ file_apprentice(struct magic_set *ms, const char *fn, int action)
 		fn = p;
 	}
 
-	efree(mfn);
```

Comment:
Without this line the file in magic_set struct contains the name of the loaded magic file.